checks/evaluation/signed_releases.go

```diff
@@ -42,7 +42,7 @@ func SignedReleases(name string, dl checker.DetailLogger, r *checker.SignedRelea
 	total := 0
 	score := 0
 	for _, release := range r.Releases {
-		if len(release.Assets) == 0 {
+		if release.ZipballURL == "" && release.TarballURL == "" && len(release.Assets) == 0 {
```

Comment:
Let's add a `log.Warn` here to say for release X, no assets or source code was found.